angular.json

```diff
@@ -121,12 +121,6 @@
                   "maximumError": "4kb"
                 }
               ],
-              "fileReplacements": [
-                {
-                  "replace": "projects/docs-tools-showcase/src/environments/environment.ts",
-                  "with": "projects/docs-tools-showcase/src/environments/environment.prod.ts"
-                }
-              ],
```

Comment:
These files are no longer needed. The `environment.prod.ts` file isn't included in `tsconfig.spec.json` and doesn't get covered in tests, and rather than accommodate it, I'm removing the environment files.